### PR TITLE
Rename to SaaS Test Tool and added README

### PR DESCRIPTION
**Pull request by arshan-alam**
Rename to SaaS Test Tool and added README

### CODE REVIEW

File: examples/saas-template/partner-integration/README.md [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+
+## Using `mpdev` to run the SaaS Test Tool
+
+
+### Before you begin
+To use `mpdev` to run the SaaS Test Tool, you must install both `Docker` and `mpdev`.
+
+1. To install Docker for Linux, follow the [Docker installation guide](https://docs.docker.com/engine/install/).
+1. To install mpdev, follow the installation instructions for [marketplace-tools](https://github.com/GoogleCloudPlatform/marketplace-tools#installation).
+1. To verify that docker can be found through your PATH environment variable, run the following command: `which docker`
```
Comment (gibbleyg) [inline examples/saas-template/partner-integration/README.md:134]: The `which` verify steps seem too obvious to be worth mentioning

**Review by gibbleyg — APPROVED** [review]

File: examples/saas-template/partner-integration/README.md (hunk 1 above)
Comment (arshan-alam) [reply examples/saas-template/partner-integration/README.md]: Thanks for the review. I've removed those steps.

**Pull request merged**